packetbeat/protos/drda/drda_mappings.go

```diff
@@ -0,0 +1,595 @@
+package drda
+
+//http://svn.apache.org/repos/asf/db/derby/code/trunk/java/client/org/apache/derby/client/net/CodePoint.java
+const DRDA_MAGIC = 0xD0
```

Comment:
C-style consts being exported. In golang prefer camelcase and use lower case if you don't want to export the symbol.